schema-engine/datamodel-renderer/src/value.rs

```diff
@@ -12,6 +12,7 @@ pub use constant::Constant;
 pub use documentation::Documentation;
 pub use env::Env;
 pub use function::{Function, FunctionParam};
+use psl::GeneratorConfigValue;
```

Comment:
Can this import be grouped with the other private imports?

Reply:
Didn't notice this at all, it's weird that rust-analyzer put it there. Thanks for catching this!